python/ngen_cal/src/ngen/cal/_hookspec.py

```diff
@@ -42,3 +44,21 @@ def ngen_cal_finish(exception: Exception | None) -> None:
     raised during the calibration loop.
     `exception` will be non-none if an exception was raised during calibration.
     """
+
+class ModelHooks():
+    @hookspec(firstresult=True)
+    def ngen_cal_model_output(id: str | None) -> Series:
+        """
+            Called during each calibration iteration to provide the model output in the form
+            of a pandas Series, indexed by time.
+            Output series should be in units of cubic meters per second.
+        """
+
+    @hookspec
+    def ngen_cal_model_post_iteration(path: Path, iteration: int) -> None:
```

Comment:
Same here:
```suggestion
    def ngen_cal_model_post_iteration(self, path: Path, iteration: int) -> None:
```

Reply:
Fixed in 082e2712272f339a94c5d01fa8a99f33a553b1ea